content/en/serverless/azure_functions/_index.md

```diff
@@ -101,7 +101,7 @@ This page explains how to collect traces, trace metrics, runtime metrics, and cu
    Datadog.Serverless.CompatibilityLayer.Start();
    ```
 
-   If your Azure Function app uses the In-Process model, add a NuGet package reference to `Microsoft.Azure.Functions.Extensions`:
+   If your Azure Function app uses the In-Container model, add a NuGet package reference to `Microsoft.Azure.Functions.Extensions`:
```

Comment:
Should we try to keep casing persistent? I see it lower and upper-cased in different areas.